frontend/src/components/common/RegistrationFooter.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
```

Comment:
IMO I don't think it makes sense to make the RegistrationFooter a common component since it's only used twice, and its particularly for the registrant experience. There's a RegistrationFooter in /RegistrantExperience/RegistrantSteps/RegistrationFooter.tsx, so I would first try to see if you can use this component with some modifications. If that's too complex, there's also a WaitlistFooter.tsx component, so use this instead. If you choose not to use the WaitlistFooter.tsx component, can you delete it, since right now I think it's just some boilerplate code.

Reply:
mmm, yeah that is a fair point. I'll make the change and use WaitlistFooter